src/libraries/System.Linq.Expressions/src/System.Linq.Expressions.csproj

```
@@ -1,22 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppCurrent)-ios;$(NetCoreAppCurrent)-tvos;$(NetCoreAppCurrent)-maccatalyst</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
```

We don't need to multi-target this project anymore since we no longer have differences between these builds.